modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -255,13 +257,18 @@ autoSaveAsDraftTest(
 		const localizableField = await page.getByRole('textbox', {
 			name: localizableFieldName,
 		});
-		const titlePlaceholder = await page.getByPlaceholder(
-			'Untitled ' + structureName
-		);
 
-		await fillAndClickOutside(page, titlePlaceholder, title);
+		await expect(async () => {
+			await fillAndClickOutside(
+				page,
+				journalEditArticlePage.titleInput,
+				title
+			);
 
-		await expect(changesSavedIndicator).toBeVisible();
+			await expect(changesSavedIndicator).toBeVisible();
+
+			await expect(undoButton).toBeEnabled();
```

Comment:
Can we only check for one thing inside `expect....toPass()`?

```js
await expect(async () => {
  // only one expect here
}).toPass();
```

I suspected that only `await expect(undoButton).toBeEnabled();` is needed.